data/data-pipeline/data_pipeline/etl/sources/census_decennial/etl.py

```diff
@@ -147,6 +148,65 @@ def __init__(self):
             field_names.CENSUS_DECENNIAL_UNEMPLOYMENT_FIELD_2009
         )
 
+        # Race/Ethnicity fields
```

Comment:
@lucasmbrown-usds I'd love your feedback on how I mapped these various variables back to the main results. I tried to follow similar logic as was in the ACS ETL, and also it's subjective so I'm happy to make any changes.